projects/ngx-auth-firebaseui/src/lib/components/ngx-auth-firebaseui/auth.component.ts

```diff
@@ -85,6 +86,7 @@ export class AuthComponent implements OnInit, AfterViewInit, OnChanges, OnDestro
   // tslint:disable-next-line:no-output-on-prefix
   @Output() onError: any;
   @Output() selectedTabChange: EventEmitter<MatTabChangeEvent> = new EventEmitter();
+  @Output() loading: EventEmitter<boolean> = new EventEmitter();
```

Comment:
why do we need a new event emitter ?

Reply:
the new event emitter was added in case we want to alert the container component something inside is loading. Can be useful to display a loader over the whole container. It doesn't disturb the flow. I can remove it though